soc/xtensa/intel_adsp/ace/power.c

```diff
@@ -179,7 +179,7 @@ void power_gate_entry(uint32_t core_id)
 	z_xt_ints_off(0xffffffff);
 }
 
-void power_gate_exit(void)
+static void __attribute__((used)) power_gate_exit(void)
```

Comment:
we have __used for `__attribute__((__used__))`

Reply:
@nashif thanks, fixed